### PR TITLE
Point to a central page explaining all signalr clients

### LINKED ISSUE #8781: ASP.NET Core SignalR Clients page on the sidebar immediately redirects to JavaScript
# Issue description

Hello. When browsing the SignalR documentation, I was confused by the fact that when accessing the sidebar and clicking "Clients", we are immediately redirected to the JavaScript client. This seems to lead to a logic where it is promoted as being 'more important than the others'. Ideally, it'd behave something like similar headers in other areas of the documentation (i.e. https://docs.microsoft.com/en-us/aspnet/web-api/overview/mobile-clients/, or the SignalR "landing page" itself https://docs.microsoft.com/en-us/aspnet/core/signalr/?view=aspnetcore-2.1).

My suggestion would be to emulate those areas, to avoid creating confusion. I have prepared a PR that attempts to address this and would appreciate advice in going through with it.

### PR DESCRIPTION
**Pull request by josemaia**
Fixes #8781

### CODE REVIEW

**Comment by scottaddie** [comment]
@josemaia Can you please sign the CLA? Once that's done, we'll review your PR.

**Comment by josemaia** [comment]
was already on it @scottaddie :)

**Comment by scottaddie** [comment]
@josemaia Please also update the link in https://github.com/aspnet/Docs/blob/master/aspnetcore/toc.md.

File: aspnetcore/toc.md [hunk 1]
```diff
@@ -219,7 +219,7 @@
 ### [HubContext](xref:signalr/hubcontext)
 ### [Users and groups](xref:signalr/groups)
 ### [Publish to Azure](xref:signalr/publish-to-azure-web-app)
-## [Clients](xref:signalr/javascript-client)
+## [Clients](xref:signalr/client)
```
Comment (scottaddie) [inline aspnetcore/toc.md]: signalr/client --> signalr/clients

**Comment by scottaddie** [comment]
@josemaia Congrats on your 1st contribution to this Docs repo! We appreciate you taking the time to fix this. Your changes will go live within the next 24 hours.

**Pull request merged**